Source/ACE/Entity/PhysicsData.cs

```diff
@@ -30,10 +30,10 @@ public class PhysicsData
         public float Friction;
         public float Elastcity;
         public uint AnimationFrame;
-        public Position Acceleration;
+        public AceVector3 Acceleration;
         public float Translucency;
-        public Position Velocity;
-        public Position Omega; // rotation
+        public AceVector3 Velocity;
+        public AceVector3 Omega; // rotation
```

Comment:
In `PhysicsData.cs` line 51, please consider if the constructor needs to have some base values:
```
      public PhysicsData()
        {
            Velocity = new AceVector3(0.0f, 0.0f, 0.0f);
            Omega = new AceVector3(0.0f, 0.0f, 0.0f);
        }
``` 

It is worth checking because I had been getting a crash where Velocity was null, when a character was logging into the server.